src/ProjectOrigin.WalletSystem.IntegrationTests/ReceiveSliceTests.cs

```diff
@@ -36,8 +36,8 @@ public async Task ReceiveSlice()
                     Registry = registryName,
                     StreamId = new Common.V1.Uuid() { Value = certId.ToString() },
                 },
-                WalletSectionPublicKey = ByteString.CopyFrom(section.PublicKey.Export()),
-                WalletSectionPosition = 2,
+                WalletDepositEndpointPublicKey = ByteString.CopyFrom(section.PublicKey.Export()),
```

Comment:
The entity stored in the database is still called "WalletSection", hence the variable `section`. Do we want to apply the rename all the way down or do we stop at the API level?

Reply:
Will be done in https://github.com/project-origin/wallet/issues/45